src/main/java/javaClasses/homework_4/Staff.java

```diff
@@ -0,0 +1,31 @@
+package javaClasses.homework_4;
+
+public abstract class Staff {
+    protected String names;
+    protected String surnames;
```

Comment:
In case of fields we always use private access modifier. Other problem is how to instantiate that field for an abstract class. In that case use super(field1, field2).